k8s/mongodb-backup-agent/container/mongodb_backup_agent_entrypoint.bash

```diff
@@ -12,9 +12,10 @@ if [[ -z "${mms_api_key}" ]]; then
 fi
 
 sed -i '/mmsApiKey/d'  $MONGODB_BACKUP_CONF_FILE
+sed -i '/mothership/d'  $MONGODB_BACKUP_CONF_FILE
 
 echo "mmsApiKey="${mms_api_key} >> $MONGODB_BACKUP_CONF_FILE
+echo "mothership=api-backup.eu-west-1.mongodb.com" >> $MONGODB_BACKUP_CONF_FILE
```

Comment:
Is this subdomain specific to us? Should it be parametrized or should we mention that they should change it (in the docs)?